src/Components/Auction/AuctionRegistrationModal.tsx

```diff
@@ -0,0 +1,93 @@
+import { Box, Button, Serif } from "@artsy/palette"
+// import { Modal } from "@artsy/palette" // not working nice with our setup yet
```

Comment:
@erikdstock - Can you describe the issue? cc @kierangillen

Reply:
Let me pull this branch down tomorrow and poke around with it, it's a good opportunity to fix any issues the new Palette button may have so if it's not urgent that'd be preferable.

Reply:
Marking resolved because we will do this upgrade once artsy/palette#554 is released.